src/Microsoft.IdentityModel.Tokens/Base64UrlEncoder.cs

```diff
@@ -76,7 +88,59 @@ public static string Encode(byte[] inArray, int offset, int length)
             if (inArray == null)
                 throw LogHelper.LogArgumentNullException("inArray");
```

Comment:
nit: I'm still not 100% on the team's style guide on this but Brent was mentioning these sorts of ArgNull checks are commonly performed with discards (_) and the null-coalescing operator (??), so:

_ = inArray ?? throw LogHelper.LogArgumentNullException("inArray");